crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -822,7 +851,7 @@ impl SpecializedMeshPipeline for MeshPipeline {
             depth_write_enabled = false;
         } else {
             label = "opaque_mesh_pipeline".into();
-            blend = Some(BlendState::REPLACE);
+            blend = None;
```

Comment:
Why the change here?

Reply:
cwfitzgerald thinks `None` will be potentially much faster in some cases, and `BlendState::REPLACE` is not needed here. I initially noticed this because in the prepass I could no longer use `BlendState::REPLACE` due to limitations on webgl.